src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -323,6 +323,12 @@ Kubernetes data is attached to these [events](/docs/using-new-relic/data/underst
   </tbody>
 </table>
 
+
+<Callout variant="important">
+  `InternalK8sCompositeSample` is an event that New Relic generates and it's quite critical for some of our interfaces. While the other events listed out in the table are billable, `InternalK8sCompositeSample` is a non-billable event. 
```

Comment:
Is it important we describe why it's critical? Do we not bill because it's a critical event? What do you think about adding something like "...is a critical event that New Relic generates when [x occurs in y environment (OR) when x occurs because of y]. This event is the only non-billable event in the table."

Reply:
Good catch! I've changed the entire callout.